assignment2/EmployeeManagement.cpp

```
@@ -0,0 +1,19 @@
+//Employee class doesn't follow srp becuase of termination and is working function
+Class Employee
+{
+    int id;
+    string name;
+    string department;
+    bool: working
+
+public:
+saveEmployeeTODatabase()
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If saveEmployeeTODatabase is a macro then please configure it.

(unknownMacro)